doc/nrf/protocols/wifi/sap_mode/mem_requirements_sap.rst

```diff
@@ -0,0 +1,16 @@
+.. _ug_wifi_mem_req_softap_mode:
+
+Memory requirements for Wi-Fi applications in SoftAP mode
+#########################################################
+
+Code and RAM memory footprint requirements differ depending on the selected platform and the application example.
```

Comment:
```suggestion
Code and RAM memory footprint requirements differ depending on the selected platform, networking application layer, and application example.
```

Reply:
Essentially, these examples differ with respect to the selected application layer...